tests/manage/monitoring/prometheus/test_noobaa.py

```diff
@@ -89,8 +87,7 @@ def test_noobaa_bucket_quota(measure_noobaa_exceed_bucket_quota):
         )
 
 
-@tier4
-@tier4a
```

Comment:
I think that this is a disruptive test that causes an error in NooBaa. It makes Namespace bucket broken and checks that proper messages are raised. I think that this qualifies as tier4a.

Reply:
tier4a is related tp day 2 operations like node replacement, device replacement kind of things. I dont think this fits in tier4a. Its neither node failure nor pod failure. Not sure if bucket deletion falls under tier4a.

Reply:
Changed into tier4a